packages/mux-player/src/video-api.ts

```diff
@@ -22,8 +22,6 @@ const AllowedVideoAttributes = {
   LOOP: 'loop',
   MUTED: 'muted',
   PLAYSINLINE: 'playsinline',
-  SRC: 'src',
```

Comment:
idea(non-blocking): I'm wondering if we should just export this to avoid the proliferation of map objects and related.